doc/guides/west/repo-tool.rst

```diff
@@ -1,79 +1,289 @@
+*********
+
+West's code is distributed via PyPI in a Python package named ``west``.
+This distribution includes a launcher executable, which is also named
+``west`` (or ``west.exe`` on Windows).
+
+When west is installed, the launcher is placed by :file:`pip3` somewhere in
+the user's filesystem (exactly where depends on the operating system, but
+should be on the ``PATH`` :ref:`environment variable <env_vars>`). This
+launcher is the command-line entry point to running both built-in commmands
+like ``west init``, ``west update``, along with any extensions discovered
+in the workspace.
+
+In addition to its command-line interface, you can also use west's Python
+APIs directly. See :ref:`west-apis` for details.
+
+.. _west-manifest-rev:
+
+The ``manifest-rev`` branch
+***************************
+
+West creates and controls a Git branch named ``manifest-rev`` in each
+project. This branch points to the revision that the manifest file
+specified for the project at the time :ref:`west-update` was last run.
+Other workspace management commands may use ``manifest-rev`` as a reference
+point for the upstream revision as of this latest update. Among other
+purposes, the ``manifest-rev`` branch allows the manifest file to use SHAs
+as project revisions.
+
+Although ``manifest-rev`` is a normal Git branch, west will recreate and/or
+reset it on the next update. For this reason, it is **dangerous**
+to check it out or otherwise modify it yourself. For instance, any commits
+you manually add to this branch may be lost the next time you run ``west
+update``. Instead, check out a local branch with another name, and either
+rebase it on top of a new ``manifest-rev``, or merge ``manifest-rev`` into
+it.
+
+.. note::
+
+   West does not create a ``manifest-rev`` branch in the manifest repository,
+   since west does not manage the manifest repository's branches or revisions.
+
+.. _west-built-in-cmds:
+
+Built-in Commands
+*****************
+
+This section gives an overview of west's built-in commands.
+
+Some commands are related to Git commands with the same name, but operate
+on the entire workspace. For example, ``west diff`` shows local changes in
+multiple Git repositories in the workspace.
+
+Some commands take projects as arguments. These arguments can be project
+names as specified in the manifest file, or (as a fallback) paths to them
+on the local file system. Omitting project arguments to commands which
+accept them (such as ``west list``, ``west forall``, etc.) usually defaults
+to using all projects in the manifest file plus the manifest repository
+itself.
+
+The following documentation does not exhaustively describe all commands.
+For additional help, run ``west <command> -h`` (e.g. ``west init -h``).
+
+.. _west-init:
+
+``west init``
+=============
+
+This command creates a west workspace. It can be used in two ways:
+
+1. Cloning a new manifest repository from a remote URL
+2. Creating a workspace around an existing local manifest repository
```

Comment:
> To me this introduction sounds like 1. and 2. are alternatives, however in the longer description below Option 2 does a subset of what Option 1 does.

They are mutually exclusive alternatives. Option 2. requires using `west init -l`, and 1. does not allow using `-l`. You'll see 1. and 2. referred to as "bootstrap" and "local" initializations in the west init source code.

Reply:
Option 1 does A+B while Option 2 performs only B. I understand Option 1 and 2 are mutually exclusive in the user interface, I was merely suggesting a different presentation focusing more on A and B than on the user interface, I think it would have been easier to read. Never mind, the current text has no ambiguity